docs/source/reference/compression/pruner.rst

```diff
@@ -11,6 +11,20 @@ Level Pruner
 
 ..  autoclass:: nni.compression.pytorch.pruning.LevelPruner
 
+.. _balanced-pruner:
+
+(experimental) Balanced Pruner
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+..  autoclass:: nni.compression.pytorch.pruning.BalancedPruner
+
+.. _block-pruner:
+
+(experimental) Block Pruner
```

Comment:
why they are experimental? what kind of changes we may apply in the future?

Reply:
As discussed, the definition of `align_n` is not clear, so `BalancedPruner` is currently the experimental stage.
For `BlockPruner`, I think `block` should be defined by a `high level mask pattern module`, this module should combine with each pruner, then no more `BlockPruner`.